src/plugins/event-plugins/ResourcePlugin.ts

```diff
@@ -120,6 +120,7 @@ export class ResourcePlugin extends InternalPlugin {
             const eventData: ResourceEvent = {
                 version: '1.0.0',
                 initiatorType: entryData.initiatorType,
+                startTime: entryData.startTime,
```

Comment:
(2) Callout: `startTime` needs to be captured in order for `ResourceEvent` to be linked to its source `PerformanceEvent`